### PR TITLE
CFY-7052 Get tenant directly from workflow context when running opera…

### PR DESCRIPTION
**Pull request by mcouthon**
…tions

This should allow using the hack during snapshot restore where we trick
the context to think it's running under an other tenant

### CODE REVIEW

File: cloudify/workflows/workflow_context.py [hunk 1]
```diff
@@ -642,6 +642,7 @@ def _build_cloudify_context(self,
             'task_name': task_name,
             'execution_id': self.execution_id,
             'workflow_id': self.workflow_id,
+            'tenant': self.tenant
```
Comment (geokala) [inline cloudify/workflows/workflow_context.py]: Trailing comma would be nice in case this needs adding to again.

Comment (mcouthon) [inline cloudify/workflows/workflow_context.py]: I personally don't like trailing commas, so I win 😝

Comment (tehasdf) [inline cloudify/workflows/workflow_context.py]: maybe comma-first as a compromise?

Comment (geokala) [inline cloudify/workflows/workflow_context.py]: The trailing commas were actually of benefit to you in this PR though, they led to it reporting the actual amount of lines you made material change to.

**Pull request merged**